src/main/java/manga/Manga.java

```diff
@@ -33,11 +47,12 @@ public boolean equals(Object o) {
 
         Manga manga = (Manga) o;
 
-        return mangaName.equals(manga.getMangaName()) && author.equals(manga.getAuthor());
+        return mangaName.equals(manga.getMangaName()) && author.equals(manga.getAuthor())
+                && deadline.equals(manga.getDeadline());
```

Comment:
Perhaps the checking of equivalent deadlines might not be necessary? I am concerned about potentially being able to add duplicate mangas from the command line since two `Manga`s are considered different if they have different deadlines. Testing is needed to confirm this.

Reply:
It would probably be safer to remove it.